src/api/__test__/getRequestHandler.test.ts

```diff
@@ -0,0 +1,18 @@
+import getRequestHandler from '../getRequestHandler';
+
+describe('get request handler should work as expected', () => {
+  const queryParams = '?id=30001';
+  const callback = jest.fn();
+  const errorCallback = jest.fn();
+  it('should fire callback function when data successfully fetched from API', async () => {
+    const url = `${process.env.REACT_APP_API || ''}/api/v2/DataItemCategoryTree`;
+    await getRequestHandler(url, queryParams, callback, errorCallback);
+    expect(callback).toHaveBeenCalled();
+  });
+
+  it('should fire error callback function when data unsuccessfully fetched from API', async () => {
+    const incorrectUrl = `${process.env.REACT_APP_API || ''}/api/v2/DataItemCategory`;
```

Comment:
Do we have a mock for this? Otherwise we might get a msw warning in the console when tests run.